tests/_utils/__init__.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Utilities and helpers for implementing the Cunumeric custom test runner.
+
+"""
+from __future__ import annotations
+
+from typing import Union
+from typing_extensions import Literal, TypeAlias
+
+FeatureType: TypeAlias = Union[
+    Literal["cpus"], Literal["cuda"], Literal["eager"], Literal["openmp"]
+]
+
+#: Value to use if --cpus is not specified.
+DEFAULT_CPUS_PER_NODE = 4
+
+#: Value to use if --gpus is not specified.
+DEFAULT_GPUS_PER_NODE = 1
+
+# TODO: (bryevdv) configurable?
+DEFAULT_GPU_MEMORY_BUDGET = 6 << 30
```

Comment:
https://github.com/nv-legate/cunumeric/pull/378/commits/1862d8760fd3d64859a222ca341f6838f0a21bd5

Reply:
Note that this value is not currently used anywhere, but it will be in the next PR. Also I made the default `--fbmem` be the same as the `6 <<30` value above, but it's not clear whether it should be that, or `4<<30` instead. Please let me know if it should be changed.

Reply:
the default for `--fbmem` should be `4<<30`, but the budget will still be 1.5X, as there are external allocations not tracked by the runtime. we need to leave some room for them.